src/pharmpy/modeling/plots.py

```diff
@@ -976,7 +975,50 @@ def vpc_plot(
     alt.Chart
         Plot
 
+
+    The simulation data should have the following format:
+
+    +-----+-------+--------+
+    | SIM | index | DV     |
+    +=====+=======+========+
+    | 1   | 0     | 0.000  |
+    +-----+-------+--------+
+    | 1   | 1     | 34.080 |
+    +-----+-------+--------+
+    | 1   | 2     | 28.858 |
+    +-----+-------+--------+
+    | 1   | 3     | 0.000  |
+    +-----+-------+--------+
+    | 1   | 4     | 12.157 |
+    +-----+-------+--------+
+    | 2   | 0     | 23.834 |
+    +-----+-------+--------+
+    | 2   | 1     | 0.000  |
+    +-----+-------+--------+
+    | ... | ...   | ...    |
+    +-----+-------+--------+
+    | 20  | 2     | 0.000  |
+    +-----+-------+--------+
+    | 20  | 3     | 31.342 |
+    +-----+-------+--------+
+    | 20  | 4     | 29.983 |
+    +-----+-------+--------+
+
+
+    Examples
+    --------
+    >>> from pharmpy.modeling import set_simulation, vpc_plot, load_example_model
+    >>> from pharmpy.tools import run_simulation
+    >>> model = load_example_model("pheno")
+    >>> sim_model = set_simulation(model, n=100)
+    >>> sim_data = run_simulation(sim_model) # doctest: +SKIP
+    >>> vpc_plot(model, sim_data) # doctest: +SKIP
     """
+    if isinstance(simulations, str) or isinstance(simulations, Path):
+        simulations = pd.read_table(simulations, delimiter=r'\s+|,', engine='python')
+        assert 'SIM' in simulations.columns and 'index' in simulations.columns
```

Comment:
I think at this level it should raise a clearer error, afaik `AssertionError` should not be raised to a user (if it does occur there is a bug). Maybe raise a `ValueError` with a clear error message.